includes/components/class-attribute.php

```diff
@@ -734,6 +734,26 @@ public function add_field_settings( $meta_box ) {
 						'html'       => true,
 						'_order'     => 120,
 					];
+
+					if('attachment_upload' === $field_type){
+						unset( $meta_box['fields'][ $field_context . '_field_default' ] );
+					}else{
+
+						// Set default field parameters.
+						$default_field_args = [
+							'label'  => hivepress()->translator->get_string( 'default' ),
+							'type'   => $field_type,
+							'_order' => 130,
+						];
+
+						if(isset( $field_settings['options'] )){
+							$default_field_args['type'] = 'number';
+							$default_field_args['description'] = esc_html__('Please put the default option ID', 'hivepress');
```

Comment:
It's ok to remove this for now, requiring setting IDs is too low-level at the moment because users generally don't know where to get the option IDs and they need simple UI. By resolving the comment above you can set a list of simple field types like number, text, date, etc. to ensure that Select, Repeater and other complex ones don't need extra logic